platform/transformations/necessary_transformations.go

```diff
@@ -0,0 +1,59 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+package transformations
+
+import (
+	"fmt"
+	"github.com/QuesmaOrg/quesma/platform/common_table"
+	"github.com/QuesmaOrg/quesma/platform/model"
+	"github.com/QuesmaOrg/quesma/platform/schema"
+	"strings"
+)
+
+// TODO it should be removed and transformations should be in 1 place.
```

Comment:
Agree with that. In fact handling terms enum should be rewritten and combined with other queries. That's just another query.

Reply:
Thanks for the remainder. I'll try to do that pretty soon. It seems more important than at least some of my ideas for gradual tech debt removal.